src/ape/contracts/base.py

```diff
@@ -414,6 +414,7 @@ def query(
         Returns:
             pd.DataFrame
         """
+        column_list = list(columns)
```

Comment:
Might need this to handle empty columns
```suggestion
        column_list = list(columns or [])
```

Reply:
`columns` is always a tuple since it is the `*args` thing and

```python
>>> list(())
[]
```

so I think this is fine.

Reply:
wonder if the data type should have just been tuple in `QueryType`

Reply:
For `_BaseQuery.columns`? Yes, it can probably be a `Tuple[...]` because after the columns list is built, it doesn't need to change in the `_BaseQuery`. However, here we are building the list from the `__fields__` so it is dynamic at this point, but it can be frozen later.

Reply:
Can think about this later lol